nvflare/app_common/utils/fl_model_utils.py

```diff
@@ -207,3 +207,15 @@ def set_meta_prop(model: FLModel, key: str, value: Any):
     @staticmethod
     def get_configs(model: FLModel) -> Optional[dict]:
         return FLModelUtils.get_meta_prop(model, MetaKey.CONFIGS)
+
+    @staticmethod
+    def update_model(model: FLModel, model_update: FLModel) -> FLModel:
+        model.meta.update(model_update.meta)
```

Comment:
Interesting question. I can see use cases for both. Since this is a library func, why don't you make it the caller's responsibility? Just add a flag "replace_meta=True", and do partial or whole update of the model's meta.

Reply:
okay. I made it replace it by default.